### PR TITLE
Edits as ELevate NG goes to stable

### PR DESCRIPTION
**Pull request by sboldyreva**

### CODE REVIEW

File: docs/elevate/ELevate-quickstart-guide.md [hunk 1]
```diff
@@ -2,112 +2,115 @@
...
+###### last updated: 2024-08-07
 
 # ELevate Quickstart Guide
 
 ::: warning
 Before beginning, we **HIGHLY** recommend that you follow system administration best practices and make sure you have backups and/or snapshots of your system before you proceed. It is recommended to do a trial run in a sandbox to verify the upgrade worked as expected before you attempt to upgrade any production system. Please report any issues encountered to the [AlmaLinux Bug Tracker](https://bugs.almalinux.org) and/or [AlmaLinux Chat Migration Channel](https://chat.almalinux.org/almalinux/channels/migration)
 :::
 
-This guide contains steps on how to upgrade your RHEL-based operating system to the next major version.
+This guide contains steps on how to upgrade your RHEL-based operating system to the next major version using the Leapp tool version **0.19.0**.
```
Comment (andrewlukoshko) [inline docs/elevate/ELevate-quickstart-guide.md:13]: leapp version is 0.16.0
leapp-repository version is 0.19.0

But it's not clear why we should mention version here, let's remove.

File: docs/elevate/ELevate-quickstart-guide.md [hunk 2]
```diff
@@ -2,112 +2,115 @@
...
-\* - upgrade to CentOS Stream 9 is currently in process and will be available later. <br>
-\** - upgrade to Oracle Linux 9 is available with the [Oracle Leapp utility](https://blogs.oracle.com/linux/post/upgrade-oracle-linux-8-to-oracle-linux-9-using-leapp) and will not be supported by ELevate project.
+\* - upgrading from Scientific Linux 7 to AlmaLinux 8 requires a workaround. Please, see more in the [known issues](#known-issues). <br>
+\** - upgrading to Oracle Linux 9 is available with the [Oracle Leapp utility](https://blogs.oracle.com/linux/post/upgrade-oracle-linux-8-to-oracle-linux-9-using-leapp) and will not be supported by ELevate project.
 
-**Requirements:** You need CentOS 7, AlmaLinux 8, EuroLinux 8 or Rocky Linux 8 system installed to use this guide.
+### Requirements 
 
-* A fully updated system is required to accomplish the upgrade. So, install the latest updates first, and reboot.
+You need CentOS 7, AlmaLinux 8, EuroLinux 8 or Rocky Linux 8 system installed to use this guide.
```
Comment (andrewlukoshko) [inline docs/elevate/ELevate-quickstart-guide.md:36]: + Scientific Linux 7
+ CentOS Stream 8

File: docs/elevate/ELevating-CentOS7-to-AlmaLinux-9.md [hunk 3]
```diff
@@ -6,10 +6,19 @@ title: "ELevating CentOS 7 to AlmaLinux 9"
 
 # ELevating CentOS 7 to AlmaLinux 9
 
-As the Leapp tool is designed to perform one-step upgrades, in order to upgrade your CentOS 7 machine to AlmaLinux 9 you need to split the upgrade process:
+This guide contains steps on how to upgrade your CentOS 7 machine to AlmaLinux OS 9 using the Leapp tool version 0.19.0.
```
Comment (andrewlukoshko) [inline docs/elevate/ELevating-CentOS7-to-AlmaLinux-9.md:9]: Don't mention leapp version

File: docs/elevate/ELevate-quickstart-guide.md (hunk 1 above)
Comment (sboldyreva) [reply docs/elevate/ELevate-quickstart-guide.md]: removed versions

File: docs/elevate/README.md [hunk 4]
```diff
@@ -14,29 +14,34 @@ The [Red Hat Upgrade Tool](https://github.com/upgrades-migrations/redhat-upgrade
...
+![image](/images/ELevateNG.svg)
+
+\* - upgrading from Scientific Linux 7 to AlmaLinux 8 requires a workaround. Please, see more in the [known issues](#known-issues). <br>
+\** - upgrading to Oracle Linux 9 is available with the [Oracle Leapp utility](https://blogs.oracle.com/linux/post/upgrade-oracle-linux-8-to-oracle-linux-9-using-leapp) and will not be supported by ELevate project.
 
 # How to upgrade
 
 Various guides cover the update steps depending on the upgrade type:
-* The [ELevate Quickstart Guide](/elevate/ELevate-quickstart-guide) covers the update steps using the Leapp utility version with 3rd party repositories support and provides the Demo video.
+* The [ELevate Quickstart Guide](/elevate/ELevate-quickstart-guide) covers the upgrade steps using the Leapp utility version 0.19.0 with 3rd party repositories support and provides the Demo video.
```
Comment (andrewlukoshko) [inline docs/elevate/README.md]: Don't mention Leapp version

File: docs/migration/README.md [hunk 5]
```diff
@@ -28,20 +28,23 @@ The [Leapp utility](https://leapp.readthedocs.io) and a few [patches](https://gi
...
 
-ELevate supports a number of 3rd party repositories for all supported systems:
-* EPEL
-* Imunify 
-* KernelCare
-* MariaDB
-* nginx
-* PostgreSQL
+:::tip 
+Please note, the ELevate project is designed to perform one-step upgrades using the Leapp tool version 0.19.0. If you wish to perform an upgrade from CentOS7, you need to split the process. Please check the [ELevating CentOS7 to AlmaLinux 9](/elevate/ELevating-CentOS7-to-AlmaLinux-9.md) guide for more information.
```
Comment (andrewlukoshko) [inline docs/migration/README.md]: Don't mention leapp version

**Review by andrewlukoshko — CHANGES_REQUESTED** [review]
Please modify according to comments

**Review by andrewlukoshko — APPROVED** [review]

**Review by bennyvasquez — APPROVED** [review]
I'll merge this in the morning in conjunction with the other work (the blog post and website updates)

**Pull request merged**